app/src/main/java/com/gravatar/demoapp/ui/components/ProfileCard.kt

```diff
@@ -36,7 +38,13 @@ fun ProfileCard(profile: UserProfile, modifier: Modifier = Modifier, avatarImage
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
         AsyncImage(
-            gravatarUrl(profile.hash ?: "", size = with(LocalDensity.current) { avatarImageSize.toPx().toInt() }),
+            AvatarUrl(Hash(profile.hash)).uri(
```

Comment:
💡 Just thinking aloud. It would be nice if I could get a `Hash` instance, or why not, an `AvatarUrl` instance directly from the `UserProfile` instance. Wdyt?

Reply:
Yes definitely, using extensions on top of the generated UserProfile code maybe?

Reply:
Sounds good!

Reply:
https://github.com/Automattic/Gravatar-SDK-Android/issues/80